taccsite_cms/static/site_cms/js/modules/setTargetForExternalLinks.js

```diff
@@ -33,13 +33,13 @@ export default function findLinksAndSetTargets(options) {
   }
 
   const links = scopeElement.getElementsByTagName('a');
+  const baseDocumentHost = document.location.host;
+  const baseDocumentHostWithWWW = `www.${baseDocumentHost}`;
```

Comment:
**Request:** Use a name that suggests relationship to the base document host and/or that this is a boolean e.g. `isSameHostAsBaseDoc…`, `baseDocSameHost…`, `hasBaseDocHost…`.

Reply:
These are not booleans yet. Ignore my dumb. 🫤